nkpremices/EPIC-Mail
# #164677325 Tests should pass even on Travis CI like locally

#### What does this PR do?
Give the good configurations of Travis 

#### Description of Task to be completed?
Have the tests of the following endpoints passing locally and on Travis with sufficient coverage :

 - POST  api/v1/auth/signup
- POST api/v1/auth/login
- POST api/v1//messages
- GET api/v1/messages

#### How should this be manually tested?
- locally : 
      After cloning the repo, run the command npm cover to see how they pass and which percentage is 
      the coverage.

- Travis : 
     Go to the git repo and see the Travis badge on the readme file.

#### What are the relevant pivotal tracker stories?
#164430194
#164441974
#164441516
